docs/development.adoc

```diff
@@ -160,33 +160,22 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ena
 
 .Procedure:
 
-To deploy an integration test:
+Integration tests can be run in the following two ways:
 
-
-* For the entire integration suite use:
-----
-$ make test-e2e
-----
-
-* For the generic tests use:
-----
-$ make test-generic
-----
-
-* For the component tests use:
+* To run the test spec parallelly (default):
++
 ----
-$ make test-cmp-e2e
+$ make clean-test
```

Comment:
why are we using "clean-test" this is fairly confusing, why not "e2e-test" ?

Reply:
Here the intention is to provide the doc reference to @mohammedzee1000 and @dharmit to write other remaining test for parallel run.